### ISSUE TITLE
The screen window will flash when switching windows on macOS ,and i recorded a video ,hope fix it

### ISSUE THREAD

**Issue by h3lp404** [isontheline/pro.webssh.net#1040]
# Bug description
_Put here a clear and concise description of what the bug is_
The screen window will flash when switching windows on macOS ,and i recorded a video ,hope fix it
# Screenshots
_If applicable, add screenshots to help explain your problem_

# Steps to reproduce
1. open the webssh
2. and open any window
3. close the it, and will flash
4. Error is thrown

https://github.com/isontheline/pro.webssh.net/assets/112472696/7be03f47-9b2d-41ac-974a-e0c4bd8353df

**Comment by h3lp404** [comment]
The issue timeout was closed earlier. Sorry.  
Now this one I recorded on video

**Comment by isontheline** [comment]
@h3lp404 Thank you for the video 🙏

Are you talking about the white flash appearing on the help page?

**Comment by h3lp404** [comment]
> @h3lp404 Thank you for the video 🙏
> 
> Are you talking about the white flash appearing on the help page?

No, it's the flicker on the left sidebar and the flicker on the top border, When I close the tab and it's flick. 
video time :0:06-0:07, and 0:08-0:10  , You can look the left sidebar and the top border  flicker .

**Comment by isontheline** [comment]
OK thanks I see it's that blank line :
![image](https://github.com/isontheline/pro.webssh.net/assets/44212923/a79f2022-7e96-486f-802b-02116dc85d80)

I will try to figure why it appears

**Comment by h3lp404** [comment]
> OK thanks I see it's that blank line : ![image](https://private-user-images.githubusercontent.com/44212923/299306544-a79f2022-7e96-486f-802b-02116dc85d80.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNDQ4NTAsIm5iZiI6MTcwNjE0NDU1MCwicGF0aCI6Ii80NDIxMjkyMy8yOTkzMDY1NDQtYTc5ZjIwMjItN2U5Ni00ODZmLTgwMmItMDIxMTZkYzg1ZDgwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDAxMDIzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzNGJjYmFhMzQ1OTI3YmQ1ZTk0YWJhNjE5NzNiMDVhODg0NzFlOGEwMDVkZmEzMTRkZjFlN2E3OGUxYmIxZDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MXEDfPxPK6useKoyz5MkvyzuROl7xSnlYIEbGdbPTeY)
> 
> I will try to figure why it appears

Great! if you use dark mode, it's clearly.
And use dark mode , it's all flicker in pic:
<img width="488" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/112472696/5111e6ce-66f3-436f-8aec-d3adbcd26f22">
